solidityContracts/transf.sol

```diff
@@ -0,0 +1,13 @@
+pragma solidity ^0.4.24;
+
+//Test contract
+contract transf {
```

Comment:
would suggest to rename it to something that reflects it being a test contract, e.g., `testcon`. Also, please add an a comment explanating what this contract does:
`// TestContract receives an uint256 and returns ....`

Reply:
Done